app/src/ui/preferences/advanced.tsx

```diff
@@ -134,32 +103,21 @@ export class Advanced extends React.Component<
           />
         </div>
         <div className="advanced-section">
-          <h2>Show a confirmation dialog before...</h2>
-          <Checkbox
-            label="Removing repositories"
-            value={
-              this.state.confirmRepositoryRemoval
-                ? CheckboxValue.On
-                : CheckboxValue.Off
-            }
-            onChange={this.onConfirmRepositoryRemovalChanged}
-          />
+          <h2>Background updates</h2>
           <Checkbox
-            label="Discarding changes"
+            label="Periodically fetch and refresh status of all repositories"
             value={
-              this.state.confirmDiscardChanges
+              this.props.repositoryIndicatorsEnabled
                 ? CheckboxValue.On
                 : CheckboxValue.Off
             }
-            onChange={this.onConfirmDiscardChangesChanged}
-          />
-          <Checkbox
-            label="Force pushing"
-            value={
-              this.state.confirmForcePush ? CheckboxValue.On : CheckboxValue.Off
-            }
-            onChange={this.onConfirmForcePushChanged}
+            onChange={this.onRepositoryIndicatorsEnabledChanged}
           />
+          <p className="git-settings-description">
+            Background updates allows Desktop to highlight repositories that
+            have uncommitted changes or are out of sync with their remote in the
+            repository list. May adversely affect performance.
```

Comment:
Not 100% sure about the "May adversely affect performance." sentence: while it's true it feels strange to have this disclaimer for a setting that's enabled by default.

I'd rather omit this until the setting is changed to be disabled by default (if that's the plan), or turn it around (e.g "Disabling this may improve performance when there are a lot of repositories to track").

Reply:
I dig it, it didn't make any difference in terms of vertical space consumed (it was still three lines) but removing the leading `This` got it down to two lines

![image](https://user-images.githubusercontent.com/634063/93607787-13da3900-f9ca-11ea-95f1-f6a1be2bf720.png)